src/osd/OSD.cc

```diff
@@ -9953,6 +9953,16 @@ void OSD::handle_conf_change(const ConfigProxy& conf,
   std::lock_guard l{osd_lock};
 
   if (changed.count("osd_max_backfills") ||
+      changed.count("osd_delete_sleep") ||
+      changed.count("osd_delete_sleep_hdd") ||
+      changed.count("osd_delete_sleep_ssd") ||
+      changed.count("osd_delete_sleep_hybrid") ||
+      changed.count("osd_snap_trim_sleep") ||
+      changed.count("osd_snap_trim_sleep_hdd") ||
+      changed.count("osd_snap_trim_sleep_ssd") ||
+      changed.count("osd_snap_trim_sleep_hybrid") ||
+      changed.count("osd_scrub_sleep") ||
+      changed.count("osd_debug_deep_scrub_sleep") ||
```

Comment:
osd_debug_deep_scrub_sleep is just for testing, no need to handle it here. @dzafman @ronen-fr it actually looks unused - is it still useful?

Reply:
Don't think so. But I was considering adding a similar option for "fairness" control.
But surely not with the 'debug' name.

Reply:
Removed osd_debug_deep_scrub_sleep based on Ronen's comments.